docs/modules/faqs.html

```diff
@@ -100,17 +100,40 @@ <h3>I have a question not addressed in this FAQ. What is my next step?</h3>
 						<h3>Can I use the Amazon Chime application to join an Amazon Chime SDK meeting?</h3>
 					</a>
 					<p>The meetings created by the Amazon Chime Application and the SDK are distinct. Amazon Chime SDK is meant for customers who would like to build the experience into their existing applications and it is <strong>NOT</strong> a means to create an Amazon Chime meeting (with a PIN) that can be joined using our client application. You cannot use the Amazon Chime SDK to build a custom client to join an Amazon Chime meeting with a PIN.</p>
+					<a href="#does-the-meetingsnotificationsconfiguration-only-support-sqs-queues-in-us-east-1" id="does-the-meetingsnotificationsconfiguration-only-support-sqs-queues-in-us-east-1" style="color: inherit; text-decoration: none;">
+						<h3>Does the MeetingsNotificationsConfiguration only support SQS queues in us-east-1?</h3>
```

Comment:
can we link to the AWS documentation in SQS or SNS  for this? Because this is not a Chime SDK platform limitation so context will help